api/management/commands/acquiredata.py

```diff
@@ -0,0 +1,29 @@
+import os
+from urllib.request import urlretrieve
+
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--path',
+            dest='path',
+            default='data',
+            help='Path to location of data files.',
+        )
+
+    def handle(self, *args, **options):
```

Comment:
Confirming that you actually want to support `*args`? Doesn't look like you use args anywhere, so why even allow specifying positional arguments?